cts/cli/regression.tools.exp

```diff
@@ -5059,6 +5059,107 @@ crm_resource: Cannot operate on clone resource instance 'promotable-rsc:0'
+=#=#=#= Begin test: Query after updating a nonexistent promotable score attribute (XML) =#=#=#=
+<pacemaker-result api-version="X" request="crm_attribute -N cluster01 -p promotable-rsc -G --output-as=xml">
+  <attribute name="master-promotable-rsc" value="1" scope="status"/>
+  <status code="0" message="OK"/>
+</pacemaker-result>
+=#=#=#= End test: Query after updating a nonexistent promotable score attribute (XML) - OK (0) =#=#=#=
+* Passed: crm_attribute  - Query after updating a nonexistent promotable score attribute (XML)
+=#=#=#= Begin test: Update an existing promotable score attribute =#=#=#=
+=#=#=#= End test: Update an existing promotable score attribute - OK (0) =#=#=#=
+* Passed: crm_attribute  - Update an existing promotable score attribute
+=#=#=#= Begin test: Update an existing promotable score attribute (XML) =#=#=#=
+<pacemaker-result api-version="X" request="crm_attribute -N cluster01 -p promotable-rsc -v 5 --output-as=xml">
+  <status code="0" message="OK"/>
+</pacemaker-result>
+=#=#=#= End test: Update an existing promotable score attribute (XML) - OK (0) =#=#=#=
+* Passed: crm_attribute  - Update an existing promotable score attribute (XML)
+=#=#=#= Begin test: Query after updating an existing promotable score attribute =#=#=#=
+scope=status  name=master-promotable-rsc value=5
+=#=#=#= End test: Query after updating an existing promotable score attribute - OK (0) =#=#=#=
+* Passed: crm_attribute  - Query after updating an existing promotable score attribute
+=#=#=#= Begin test: Query after updating an existing promotable score attribute (XML) =#=#=#=
+<pacemaker-result api-version="X" request="crm_attribute -N cluster01 -p promotable-rsc -G --output-as=xml">
+  <attribute name="master-promotable-rsc" value="5" scope="status"/>
+  <status code="0" message="OK"/>
+</pacemaker-result>
+=#=#=#= End test: Query after updating an existing promotable score attribute (XML) - OK (0) =#=#=#=
+* Passed: crm_attribute  - Query after updating an existing promotable score attribute (XML)
+=#=#=#= Begin test: Delete an existing promotable score attribute =#=#=#=
+Deleted status attribute: id=status-1-master-promotable-rsc name=master-promotable-rsc
+=#=#=#= End test: Delete an existing promotable score attribute - OK (0) =#=#=#=
+* Passed: crm_attribute  - Delete an existing promotable score attribute
+=#=#=#= Begin test: Delete an existing promotable score attribute (XML) =#=#=#=
+<pacemaker-result api-version="X" request="crm_attribute -N cluster01 -p promotable-rsc -D --output-as=xml">
+  <status code="0" message="OK"/>
+</pacemaker-result>
+=#=#=#= End test: Delete an existing promotable score attribute (XML) - OK (0) =#=#=#=
+* Passed: crm_attribute  - Delete an existing promotable score attribute (XML)
+=#=#=#= Begin test: Query after deleting an existing promotable score attribute =#=#=#=
+crm_attribute: Error performing operation: No such device or address
```

Comment:
FYI, one catch in regression testing is that any output that comes from a system library (like the errno message above) can vary from OS to OS. Another one is output based on local conditions (like a timestamp). Our general approach to such cases is that huge last sed in the file, which is hacky but works since these don't usually change. Once CI completes we'll know if there are any such cases that need to be dealt with ...

Reply:
Ouch, noted.

Reply:
Hooray, all good except the rawhide build (as usual).